README.md

```diff
@@ -0,0 +1,113 @@
+# Action Server
+The action server is an actionlib server for managing the execution of high
+level tasks and their semantic chaining. It takes actionlib goals describing
+series of high level tasks in json form, checks the consistency of the
```

Comment:
My first reaction was: "Why JSON" embedded in a ROS message" I know it's because the variability of the structure of the commands, but some explaining of this can't hurt. Not directly but maybe at the end of this document.

Reply:
Check :)